src/interfaces/index.ts

```diff
@@ -74,3 +74,27 @@ export interface State {
     isOpen: boolean;
   };
 }
+
+export interface IChartProps {
+  dataSet: {
+    date: string;
+    confirmed: number;
+    deaths: number;
+    recovered: number;
+    confirmed_diff: number;
+    deaths_diff: number;
+    recovered_diff: number;
+    last_update: string;
+    active: number;
+    active_diff: number;
+    fatality_rate: number;
+    region: {
+      iso: string;
+      name: string;
+      province: string;
+      lat: string;
+      long: string;
+      cities: never[];
```

Comment:
I dont use that in my type, because in everytime is void, you can delete that

Reply:
The part of the cities.

Reply:
remove never